docs/account_management/xias/index.md

```diff
@@ -12,3 +12,18 @@ An overview of creating an external collaborator account:
 2. The guest must then [Create a XIAS Guest Account](guest_instructions.md)
 
 We recommend the sponsor and guest stay in close contact during the process in case anything unexpected occurs. If you encounter difficulties with any part of the XIAS website process, please contact [AskIT](mailto:askit@uab.edu). If you encounter difficulties with the Cheaha account creation portion at the Open OnDemand web portal, please contact [Support](../../help/support.md)
+
+## What is next and how to login to services?
+
+To login to the [Cheaha](../../cheaha/getting_started.md) and [GitLab](../gitlab_account.md) services, you need to use your XIAS email address and the password used to create your XIAS account. To access the Cloud.rc, you first need to request an account. Please follow [Cloud.rc](../../uab_cloud/index.md#first-steps) to initiate the request.
+ <!-- your username is your XIAS email address, while the password can be any current password you have, not necessarily the same as the one used to create your XIAS email address. -->
```

Comment:
Add blank line before comments and new paragraphs.

Reply:
Line addition fixed.